xmlWorksheet.go

```diff
@@ -462,9 +462,10 @@ type xlsxC struct {
 	R        string   `xml:"r,attr,omitempty"` // Cell ID, e.g. A1
 	S        int      `xml:"s,attr,omitempty"` // Style reference.
 	// Str string `xml:"str,attr,omitempty"` // Style reference.
-	T  string  `xml:"t,attr,omitempty"` // Type.
-	F  *xlsxF  `xml:"f,omitempty"`      // Formula
-	V  string  `xml:"v,omitempty"`      // Value
+	T  string  `xml:"t,attr,omitempty"`  // Type.
+	T2 string  `xml:"t2,attr,omitempty"` // Type2. (richtext)
```

Comment:
The field `T2` is not defined in the ECMA-376, ISO/IEC 29500 spec, and it doesn't have a good semanticization, I don't think this fixed in the right way.

Reply:
got it, I will try other